rustual-boy-core/src/v810.rs

```diff
@@ -993,15 +993,24 @@ impl V810 {
             return;
         }
 
-        self.enter_exception(exception_code);
+        let interrupt_level = (exception_code as usize >> 4) & 0x0f;
```

Comment:
nit: extra parens around casts are preferred, esp. when using shifts etc, just to make the precedence a bit more clear :)